### PR TITLE
Issue 19627: Update MP JWT to wait for config proxy services to start before activating

### PR DESCRIPTION
**Pull request by ayoho**
For #19627

### CODE REVIEW

**Comment by ayoho** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.security.mp.jwt.1.1_fat.mpJwt-1.1,com.ibm.ws.security.mp.jwt.1.1_fat.mpJwt-1.2,com.ibm.ws.security.mp.jwt.1.1_fat.mpJwt-2.0,com.ibm.ws.security.mp.jwt.1.2_fat.mpJwt-1.2,com.ibm.ws.security.mp.jwt.1.2_fat.mpJwt-2.0

**Comment by ayoho** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.security.mp.jwt.1.1_fat.mpJwt-1.1,com.ibm.ws.security.mp.jwt.1.1_fat.mpJwt-1.2,com.ibm.ws.security.mp.jwt.1.1_fat.mpJwt-2.0,com.ibm.ws.security.mp.jwt.1.2_fat.mpJwt-1.2,com.ibm.ws.security.mp.jwt.1.2_fat.mpJwt-2.0

File: dev/com.ibm.ws.security.mp.jwt/src/com/ibm/ws/security/mp/jwt/config/MpConfigUtil.java [hunk 1]
```diff
@@ -15,30 +15,35 @@
...
+@Component(property = { "service.vendor=IBM" })
 public class MpConfigUtil {
     private static TraceComponent tc = Tr.register(MpConfigUtil.class, TraceConstants.TRACE_GROUP, TraceConstants.MESSAGE_BUNDLE);
-    private final AtomicServiceReference<MpConfigProxyService> mpConfigProxyServiceRef;
 
-    public MpConfigUtil(AtomicServiceReference<MpConfigProxyService> mpConfigProxyServiceRef) {
-        this.mpConfigProxyServiceRef = mpConfigProxyServiceRef;
+    private static MpConfigProxyService mpConfigProxyService;
+
+    @Reference
```
Comment (brenthdaniel) [inline dev/com.ibm.ws.security.mp.jwt/src/com/ibm/ws/security/mp/jwt/config/MpConfigUtil.java]: @Reference(cardinality=ReferenceCardinality.AT_LEAST_ONE, policy=ReferencePolicy.DYNAMIC)

**Comment by ayoho** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.security.mp.jwt.1.1_fat.mpJwt-1.1,com.ibm.ws.security.mp.jwt.1.1_fat.mpJwt-1.2,com.ibm.ws.security.mp.jwt.1.1_fat.mpJwt-2.0,com.ibm.ws.security.mp.jwt.1.2_fat.mpJwt-1.2,com.ibm.ws.security.mp.jwt.1.2_fat.mpJwt-2.0

**Comment by ayoho** [comment]
#libby
#build
#spawn.fullfat.buckets=com.ibm.ws.security.mp.jwt.1.1_fat.mpJwt-1.1,com.ibm.ws.security.mp.jwt.1.1_fat.mpJwt-1.2,com.ibm.ws.security.mp.jwt.1.1_fat.mpJwt-2.0,com.ibm.ws.security.mp.jwt.1.2_fat.mpJwt-1.2,com.ibm.ws.security.mp.jwt.1.2_fat.mpJwt-2.0

**Review by teddyjtorres — APPROVED** [review]

**Pull request merged**